summit.html

```diff
@@ -38,8 +38,11 @@ <h1 class="section-heading">The practicals</h1>
               Who: To get in touch with the organizers send an email to
               <a href="mailto:eiffel-tc@googlegroups.com">eiffel-tc@googlegroups.com</a>.
             </li>
-	  </ul>
-	</p>
+            <li>
+              Questions: To ask questions during, after or before the summit please join us on <a href="https://eiffel-workspace.slack.com/archives/C02EKUS4M24">slack</a>.
```

Comment:
Well, we don't have to limit ourselves to questions.

Reply:
If we want to capture not just questions, would the following work?

```suggestion
              Communication: For questions, comments, or suggestions regarding the Summit, please join us on <a href="https://eiffel-workspace.slack.com/archives/C02EKUS4M24">slack</a>.
```

Reply:
Sure, that'd work.